kdumpctl

```diff
@@ -1030,6 +1032,53 @@ check_final_action_config()
 	esac
 }
 
+LUKS_SYSFS=/sys/kernel/crash_dm_crypt_keys
+reuse_loaded_luks_keys()
+{
+	local _luks_dev _count _cmds _cmd _state
+	declare -a _luks_devs
+
+	_state=$(< $LUKS_SYSFS)
+
+	if [[ $_state == loaded ]]; then
+		echo -n reuse > $LUKS_SYSFS
+		return 0
+	fi
+}
+
+prepare_luks()
+{
+	local _luks_dev _count _cmds _cmd _state
+	declare -a _luks_devs
+
+	_state=$(< $LUKS_SYSFS)
```

Comment:
What happens when `$LUKS_SYSFS` doesn't exist? E.g. when you are running on an old kernel or one where the required configs aren't set?